### PR TITLE
Improvement/docker compose terraform

### PR DESCRIPTION
**Pull request by aboubakar14**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    tty: true
+    image: ollama/ollama:latest
+    ports:
+      - 11434:11434
+    networks:
+      - ollama-docker
+    deploy:
+      resources:
+        reservations:
+          devices:
```
Comment (Zorin95670) [inline docker-compose.yaml]: after reading your PR, it's the compose only for nvdia
so rename it to `docker-compose.nvdia.yml`
And add one for non nvidia

File: README.md [hunk 2]
```diff
@@ -187,9 +105,108 @@ Gemini has the following settings:
...
+
+```sh
+hypercorn src.main:app --reload --bind 127.0.0.1:8585
+```
+
+Once it is running, you can request it on this url: ```http://localhost:8585/```
+
+And the Swagger UI is available on this url: ```http://localhost:8585/docs```
+
+## How to launch the API using docker
```
Comment (Zorin95670) [inline README.md]: add to part in it, one for nvidia one for another

File: docker-compose.yaml (hunk 1 above)
Comment (aboubakar14) [reply docker-compose.yaml]: done

File: README.md (hunk 2 above)
Comment (aboubakar14) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -187,8 +105,121 @@ Gemini has the following settings:
...
+
+```sh
+hypercorn src.main:app --reload --bind 127.0.0.1:8585
+```
+
+Once it is running, you can request it on this url: ```http://localhost:8585/```
+
+And the Swagger UI is available on this url: ```http://localhost:8585/docs```
+
+## How to launch the API using docker (NVIDIA GPU)
```
Comment (Zorin95670) [inline README.md]: prefere to have:
```
## How to launch the API using docker

### Without nvidia GPU

### With nvidia GPU
```

File: src/handlers/Ollama/ModelFiles/generate/default-terraform-mistral-modelfile_generate [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        "name": "terraform.tf",
+        "content": "provider \"aws\" {\n  region = \"us-west-2\"\n}\n\nresource \"aws_instance\" \"example\" {\n  ami           = \"ami-0c55b159cbfafe1f0\"\n  instance_type = \"t2.micro\"\n\n  tags = {\n    Name = \"ExampleInstance\"\n  }\n}"
+    }
+]
+```
+
+If there are several files, each file must be in another item.
+I only want the code, in json as specified ealier, and nothing more. No explanation, nothing more.
+"""
```
Comment (Zorin95670) [inline src/handlers/Ollama/ModelFiles/generate/default-terraform-mistral-modelfile_generate]: missing end of line in some files, please add it

File: README.md (hunk 3 above)
Comment (aboubakar14) [reply README.md]: done

File: src/handlers/Ollama/ModelFiles/generate/default-terraform-mistral-modelfile_generate (hunk 4 above)
Comment (aboubakar14) [reply src/handlers/Ollama/ModelFiles/generate/default-terraform-mistral-modelfile_generate]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**